### PR TITLE
Dz feat progress bar

### PR DESCRIPTION
**Pull request by yosephhabtamu**

### CODE REVIEW

File: src/app/shared/shared-directives/button-interaction.directive.spec.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// import { ButtonInteractionDirective } from './button-interaction.directive';
+
+// describe('ButtonInteractionDirective', () => {
+//   it('should create an instance', () => {
+//     const directive = new ButtonInteractionDirective();
+//     expect(directive).toBeTruthy();
```
Comment (destifo) [inline src/app/shared/shared-directives/button-interaction.directive.spec.ts:6]: why is it commented?

**Review by destifo — CHANGES_REQUESTED** [review]
address the comment

File: src/app/shared/shared-directives/button-interaction.directive.spec.ts (hunk 1 above)
Comment (yosephhabtamu) [reply src/app/shared/shared-directives/button-interaction.directive.spec.ts]: It was some test for the directive. The directive is not working so I removed both (the directive and test ofcourse)

Comment (destifo) [reply src/app/shared/shared-directives/button-interaction.directive.spec.ts]: ok.

**Review by destifo — APPROVED** [review]

**Pull request merged**